### ISSUE TITLE
[webview_flutter] Support File interface

### ISSUE THREAD

**Issue by bbrto21** [flutter-tizen/plugins#222]
A problem has been reported that a specific page for in-house app cannot be loaded, because `webview_flutter` does not support the [File interface](https://w3c.github.io/FileAPI/#file-section).

**log**
```bash
I/Starfish(54023): 1.0.0/: will fire top browsing context Window.ON-LOAD event
I/ConsoleMessage(54023): flutter: Page finished loading: http://10.42.0.139/api-test
E/Starfish(54023): 1.0.0/: console.error: TypeError: Invalid operand to 'instanceof': right expr is not a function object
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:59828
E/Starfish(54023): tion xe(e){var t,n;if(ye(e))return e;if(e instanceof se.a.File)return e;if(!_e(e
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:415653
E/Starfish(54023): (n=["resolvedSubtrees"]).call(n,s()(i)),Object(g.i)(o))})),i()(r,x.UPDATE_PARAM,
E/Starfish(54023):                                         ^
E/Starfish(54023): at function call() { [native function] } 
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:968192
E/Starfish(54023): n),i=0;i<n;i++)o[i]=arguments[i];return e.call.apply(e,u()(t=[this]).call(t,o))}
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:963220
E/Starfish(54023): !e)return t;var r=e[n.type];if(r){var o=Z(r)(t,n);return null===o?t:o}return t}}
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:680390
E/Starfish(54023): unction(t){n.forEach((function(n){var r=(0,e[n])(t.get(n),o);(0,a.validateNextSt
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:680361
E/Starfish(54023): [1];return r.withMutations((function(t){n.forEach((function(n){var r=(0,e[n])(t.
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:15644
E/Starfish(54023): nction(e){var t=this.asMutable();return e(t),t.wasAltered()?t.__ensureOwner(this
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:680332
E/Starfish(54023): ?arguments[0]:t(),o=arguments[1];return r.withMutations((function(t){n.forEach((
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:217259
E/Starfish(54023):  may not dispatch actions.");try{u=!0,i=o(i,e)}finally{u=!1}for(var t=a=s,n=0;n<
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:61076
E/Starfish(54023): on(n){return"function"==typeof n?n(e()):t(n)}}}}function Te(e){var t,n=e.keySeq(
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:219444
E/Starfish(54023): unction D(e,t){return function(){return t(e.apply(void 0,arguments))}}function O
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:89295
E/Starfish(54023): e.catch(11),console.error(e.t0);case 22:p.updateResolvedSubtree([],k.resultMap);
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:779951
E/Starfish(54023): n l(e,t,n){try{return{type:"normal",arg:e.call(t,n)}}catch(e){return{type:"throw
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:779741
E/Starfish(54023): hod&&n.abrupt("return",n.arg);r=h;var u=l(e,t,n);if("normal"===u.type){if(r=n.do
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:780373
E/Starfish(54023): ((function(t){u(e,t,(function(e){return this._invoke(t,e)}))}))}function S(e,t){
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:126146
E/Starfish(54023): 99);function o(e,t,n,o,i,a,s){try{var u=e[a](s),c=u.value}catch(e){return void n
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:126349
E/Starfish(54023): n(r,i){var a=e.apply(t,n);function s(e){o(a,r,i,s,u,"next",e)}function u(e){o(a,
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:774368
E/Starfish(54023): rejection=1),!0===l?a=r:(h&&h.enter(),a=l(r),h&&(h.exit(),u=!0)),a===c.promise?f
E/Starfish(54023):                                         ^
E/Starfish(54023): at http://10.42.0.139/js/chunk-4f43e03e.991b370e.js:3:777426
E/Starfish(54023): main)&&e.exit();o;){t=o.fn,o=o.next;try{t()}catch(e){throw o?a():i=void 0,e}}i=v
E/Starfish(54023):                                         ^
E/ConsoleMessage(54023): GBM_ERR::get_gpu_pxl_fmt(196):::No map for format: 0x34325241
E/ConsoleMessage(54023): GBM_ERR::get_gpu_pxl_fmt(196):::No map for format: 0x34325241
I/E20_MONITOR(55486): ## Enlightenment (PID=517, STAT=S, WCHAN=do_select) is running ! ##
W/LAUNCHPAD(  716): launchpad_memory_monitor.c: _memory_monitor_is_low_memory(104) > previous used ratio(39), current used ratio(40)
^C​
```
the original issue was reported by @Juwols

**Comment by bwikbs** [comment]
@bbrto21 What interface do we need? (specifically) Are you working on this?

**Comment by bbrto21** [comment]
Specifically, it needs `File` interface that exposes to `window` object. however, it is not yet known which properties of this interface are actually used. I haven't started anything, you can start this if you're interested in this.

**Comment by bwikbs** [comment]
Do you know how to reproduce this issue?

**Comment by bbrto21** [comment]
> Do you know how to reproduce this issue?

I will contact you internally.

**Comment by bbrto21** [comment]
Assertion about unimplemented feature in LineFormattingContext::computeVerticalProperties

**Comment by swift-kim** [comment]
Has this issue been resolved by #227?
